
[ Fri Mar  1 10:45:36 PM CET 2024 ] 



[ Fri Mar  1 10:46:12 PM CET 2024 ] 



[ Fri Mar  1 10:47:10 PM CET 2024 ] 

diff --git a/home/roman/backup/cache/configuration.nix b/home/roman/backup/backup/configuration.nix
index 7b13b3b..ae3deec 100644
--- a/home/roman/backup/cache/configuration.nix
+++ b/home/roman/backup/backup/configuration.nix
@@ -41,7 +41,7 @@
   # Configure keymap in X11
   services.xserver = {
     layout = "us";
-    xkbVariant = "";
+    xkb.variant = "";
 
     # Nvidia
     videoDrivers = [ "nvidia" ];

[ Fri Mar  1 10:47:25 PM CET 2024 ] 

diff --git a/home/roman/backup/cache/configuration.nix b/home/roman/backup/backup/configuration.nix
index ae3deec..cd41ca9 100644
--- a/home/roman/backup/cache/configuration.nix
+++ b/home/roman/backup/backup/configuration.nix
@@ -40,7 +40,7 @@
 
   # Configure keymap in X11
   services.xserver = {
-    layout = "us";
+    xkb.layout = "us";
     xkb.variant = "";
 
     # Nvidia

Diff:






